src/main/kotlin/com/example/goodtripserver/trip/route/TripController.kt

```diff
@@ -0,0 +1,85 @@
+package com.example.goodtripserver.trip.route
+
+import com.example.goodtripserver.trip.model.AddCountryRequest
+import com.example.goodtripserver.trip.model.AddNoteRequest
+import com.example.goodtripserver.trip.model.AddTripRequest
+import com.example.goodtripserver.trip.service.TripService
+import com.goodtrip.goodtripserver.database.models.Trip
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.DeleteMapping
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.ResponseBody
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@RequestMapping("/trip")
+class TripController {
```

Comment:
In general I dont understand you you wrap the data into ResponseEntity manually, you can just return the classes that you need as I know, Spring should do it for yourself

Reply:
https://spring.io/guides/gs/rest-service